### PR TITLE
IEEE 17 GET: /api/hardware/orders/

### PR DESCRIPTION
**Pull request by EdwardJTL**
## Overview

- Resolves IEEE-17
- implement GET: /api/hardware/orders/
- Add serializer for Hardware Order model
- Add list view for GET requests
- Add url for api endpoint
- Add test case for serializer and GET view

## Unit Tests Created

- Serializer test to test empty order and order with 2 hardware
- GET api test for authentication and 2 orders with 2 hardware each

## Steps to QA

- No migration needed. Just run the test cases.

### CODE REVIEW

File: hackathon_site/hardware/serializers.py
Comment (grahamhoyes) [inline hackathon_site/hardware/serializers.py]: This can just be called `OrderSerializer` to stick with our convention of naming it after the model

**Review by grahamhoyes — COMMENTED** [review]
Just a rename on the serializer, everything else LGTM

File: hackathon_site/hardware/serializers.py
Comment (EdwardJTL) [reply hackathon_site/hardware/serializers.py]: renamed. also renamed the tests

File: hackathon_site/hardware/views.py
Comment (grahamhoyes) [inline hackathon_site/hardware/views.py]: This one too! `OrderListView`

Comment (EdwardJTL) [reply hackathon_site/hardware/views.py]: renamed :)

**Review by grahamhoyes — APPROVED** [review]
Good stuff :+1: 

Before you merge, click the dropdown next to the merge button and change it to squash and merge.

**Pull request merged**
